### ISSUE TITLE
Wrong icons?

### ISSUE THREAD

**Issue by Gabriel-p** [rbribeiro/wingpanel-indicator-dropbox#18]
The same 'X' icon is shown at all times. Is this a bug or a feature that hasn't been completed yet?

**Comment by rbribeiro** [comment]
It should show three different icons depending on the status of the dropbox service: syncing, idle, paused

**Comment by Gabriel-p** [comment]
Nope, I only see the X icon at all times. Is there a log or some debugging steps I can do to help with this issue?

**Comment by pefferen** [comment]
same here, running Elementary 7. I do get a deprecation message from the officiall Dropbox cli client, this could be related:

````bash
$ dropbox status
/usr/bin/dropbox:614: DeprecationWarning: isSet() is deprecated, use is_set() instead
  if self.stop_event.isSet(): break
Updated
````
